jena-java/src/main/java/eu/neverblink/jelly/convert/jena/JenaAdapters.java

```diff
@@ -0,0 +1,122 @@
+package eu.neverblink.jelly.convert.jena;
+
+import eu.neverblink.jelly.core.EncodedNamespaceDeclaration;
+import eu.neverblink.jelly.core.GraphDeclaration;
+import eu.neverblink.jelly.core.utils.DatasetAdapter;
+import eu.neverblink.jelly.core.utils.GraphAdapter;
+import eu.neverblink.jelly.core.utils.IteratorUtils;
+import java.util.Collections;
+import org.apache.jena.graph.Graph;
+import org.apache.jena.graph.Node;
+import org.apache.jena.graph.NodeFactory;
+import org.apache.jena.graph.Triple;
+import org.apache.jena.query.Dataset;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.sparql.core.DatasetGraph;
+import org.apache.jena.sparql.core.Quad;
+
+/**
+ * Adapters for Jena structures.
+ * <p>
+ * These adapters are used to convert Jena structures to triples and quads.
+ * <p>
+ * Warning: the adapters assume that the underlying structures are static!
+ * If they change during iteration, the results are undefined.
+ */
+public final class JenaAdapters {
```

Comment:
Add a comment explaining that this is mostly for the integration with the stream module.